src/components/TappableCopy.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import {View, Pressable, Linking} from 'react-native';
+import PropTypes from 'prop-types';
+import styles from '../styles/styles';
+import compose from '../libs/compose';
+import {Checkmark, Clipboard as ClipboardIcon} from './Icon/Expensicons';
+import Clipboard from '../libs/Clipboard';
+import ContextMenuItem from './ContextMenuItem';
+import withLocalize, {withLocalizePropTypes} from './withLocalize';
+import withWindowDimensions, {windowDimensionsPropTypes} from './withWindowDimensions';
+import CONST from '../CONST';
+
+const propTypes = {
+    /** Children to wrap in TappableCopy. */
+    children: PropTypes.node.isRequired,
+
+    /** Styles to be assigned to Container */
+    style: PropTypes.arrayOf(PropTypes.object),
+
+    /** Decides Tap behaviour. */
+    type: PropTypes.oneOf([CONST.LOGIN_TYPE.PHONE, CONST.LOGIN_TYPE.EMAIL]),
+
+    /** Value to be copied or passed via tap. */
+    value: PropTypes.string.isRequired,
+
+    ...windowDimensionsPropTypes,
+    ...withLocalizePropTypes,
+};
+
+const defaultProps = {
+    style: [],
+    type: undefined,
```

Comment:
I don't quite understand this. When are we using this component and why would it not have a `type`? It looks like if there's no `type` we just return an empty `View`. 🤔 

Can we maybe give this a better name than `TappableCopy`? It looks like it has a pretty specific purpose i.e. wrap _anything_ (I don't see anything that limits this to "copy") in a `Pressable` and then do an email or phone link or generically copy the text to clipboard.

Reply:
Maybe something like `CommunicationsLink`

Reply:
Sure.

Reply:
There were two points to address so I'm reopening. Please allow reviewers to resolve comments.